app/controllers/thredded/autocomplete_users_controller.rb

```diff
@@ -6,17 +6,27 @@ class AutocompleteUsersController < Thredded::ApplicationController
     def index
       authorize_creating PrivateTopicForm.new(user: thredded_current_user).private_topic
       users = params.key?(:q) ? users_by_prefix : users_by_ids
+
       render json: {
-        results: users.map do |user|
-          { id:         user.id,
-            name:       user.send(Thredded.user_name_column),
-            avatar_url: Thredded.avatar_url.call(user) }
-        end
+          results: users.map do |user|
+            { id:         user.id,
+              name:       display_name(user),
+              avatar_url: Thredded.avatar_url.call(user) }
+          end
       }
     end
 
     private
 
+    # Defines how the name is displayed in the autocomplete search results
+    def display_name(user)
+      if Thredded.user_display_name_method == :to_s
```

Comment:
I'm fine with adding the ability to customize this, but this check should either be `Thredded.user_display_name_method == Thredded.user_name_column`, or alternatively no check just:

```ruby
[user.send(Thredded.user_name_column), user.send(Thredded.user_display_name_method)].uniq.join(' - ')
```